modules/ai/module.php

```diff
@@ -24,22 +37,31 @@ public function __construct() {
 		} );
 
 		add_action( 'elementor/ajax/register_actions', function( $ajax ) {
-			$ajax->register_ajax_action( 'ai_get_user_information', [ $this, 'ajax_ai_get_user_information' ] );
```

Comment:
Completely optional change. Just thought it makes sense to not repeat `$ajax->register_ajax_action` every time we register a handler 🙂